src/ol_concourse/pipelines/container_images/jupyter_courses.py

```diff
@@ -44,6 +46,41 @@ class CourseImageInfo:
         repo_uri="git@github.mit.edu:ol-notebooks-qa/course_clustering_and_descriptive_ai.git",
         image_name="clustering_and_descriptive_ai",
     ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.6-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.6-3T2025.git",
+        image_name="UAI_SOURCE-UAI.6-3T2025",
+    ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.7-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.7-3T2025.git",
+        image_name="UAI_SOURCE-UAI.7-3T2025",
+    ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.8-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.8-3T2025.git",
+        image_name="UAI_SOURCE-UAI.8-3T2025",
+    ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.9-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.9-3T2025.git",
+        image_name="UAI_SOURCE-UAI.9-3T2025",
+    ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.11-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.11-3T2025.git",
+        image_name="UAI_SOURCE-UAI.11-3T2025",
+    ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.12-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.12-3T2025.git",
+        image_name="UAI_SOURCE-UAI.12-3T2025",
+    ),
+    CourseImageInfo(
+        course_name="UAI_SOURCE-UAI.13-3T2025",
+        repo_uri="git@github.mit.edu:ol-notebooks-qa/UAI_SOURCE-UAI.13-3T2025.git",
```

Comment:
Did we change the org name yet? Probably worth doing and updating all of the images in this before we go much further.

Reply:
I haven't renamed it yet - I figured I'd do so once the initial crop of these images were built and in ECR. 

Minor lift either way since this is the only place that cares about the org name at the moment, but it is on my list to do!